07-studio/00-overview.md

```diff
@@ -0,0 +1,28 @@
+---
+pageTitle: TypeDB Studio
+keywords: typeql, studio , visualiser, visualisation, visualizer, visualization
+longTailKeywords: typedb studio, typeql studio, typeql visualiser, typeql visualisation, typeql visualizer, typeql visualization
+Summary: An overview of TypeDB Studio.
+toc: false
+---
+## What is Studio?
+![Connection](/docs/images/studio/studio.png)
+[caption: TypeDB Studio, a dedicated IDE for TypeDB]
+
+The TypeDB Studio, along with the [TypeDB Console](../02-console/01-console.md) and the [TypeDB Clients](../03-client-api/00-overview.md), is an interface through which we can read from a TypeDB database.
+Studio allows us to execute any [TypeQL query](../11-query/00-overview.md), and visualise and investigate its results.
+Whether you need a tool to test and experiment with your newly created TypeDB knowledge graph, or that you prefer a graphical interface for reading data from TypeDB, you will find Studio extremely useful.
```

Comment:
Although the docs are not consistently up-to-date in this regard, I think we should favour the term "database"
```suggestion
Whether you need a tool to test and experiment with your newly created TypeDB database, or you prefer a graphical interface for reading data from TypeDB, you will find Studio extremely useful.
```